src/mpi_native/mpi_native.cpp

```diff
@@ -235,7 +235,6 @@ int MPI_Bcast(void* buffer,
         world.broadcast(
           rank, (uint8_t*)buffer, datatype, count, faabric::MPIMessage::NORMAL);
     } else {
-        auto fptr = &faabric::scheduler::MpiWorld::recv;
```

Comment:
This variable was unused, and adding the compiler checks to `faabric` raised the issue.